twitterscraper/tweet.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, user, fullname, id, url, timestamp, text, replies, retweets,
         self.retweets = retweets
         self.likes = likes
         self.html = html
+        self.reply_to_id = reply_to_id
```

Comment:
`self.reply_to_id = 0 if id == reply_to_id else reply_to_id` 
sets it to zero if it is equal to the tweet-id, i.e. if it is not a reply to anyone. Giving the reply_to_id a value even when it is not a reply is misleading and people would have to check the equivalence of id and reply_to_id before they can be sure it is an reply.